internal/pkg/describe/webapp.go

```diff
@@ -111,7 +111,7 @@ type ecsService interface {
 
 func (uri *WebAppURI) String() string {
 	if uri.Path != "" {
-		return fmt.Sprintf("%s and path %s", color.HighlightResource("http://"+uri.DNSName), color.HighlightResource(uri.Path))
+		return fmt.Sprintf("%s/%s", color.HighlightResource("http://"+uri.DNSName), color.HighlightResource(uri.Path))
```

Comment:
I think we'll have to remove the colorize here since it's now being used in AppShow for JSON output. Otherwise, you'll get a bunch of control characters in the output for the colors. This is very sneaky - the end to end tests would have passed, but that's because we disable color.

Reply:
Sounds good, maybe in general all other packages except `cli` should not colorize text. That will make it easier to control when data should be colored. 
We can move the coloring instead to: 
https://github.com/aws/amazon-ecs-cli-v2/blob/6aa1f82610f349620b6e9333d2e85cfaa501f8ed/internal/pkg/cli/app_deploy.go#L441

Reply:
Removed colorize and made the change in app_deploy.